src/app/routes/onDemandRadio/getInitialData/index.js

```diff
@@ -82,6 +91,11 @@ export default async ({ path: pathname, pageType, service, toggles }) => {
         ),
         episodeAvailability: getEpisodeAvailability(json),
         radioScheduleData: getRadioScheduleData(json),
+        recentEpisodes: processRecentEpisodes(json, {
+          exclude: episodeId,
+          enabled,
```

Comment:
I think boolean arguments in functions can be a bit of code smell because it declares that the function does more one thing. You could not pass this in and instead check whether to call the function using a ternary or conditional object property. I did a quick google on boolean args and found some articles e.g. https://understandlegacycode.com/blog/what-is-wrong-with-boolean-parameters/